Scripts/Editor/Configuration/WitConfigurationEditor.cs

```diff
@@ -20,65 +20,111 @@ public class WitConfigurationEditor : Editor
 
     private int selectedToolPanel;
 
-    private readonly string[] toolPanelNames = new []
+    private readonly string[] toolPanelNames = new[]
     {
+        "Application",
         "Intents",
         "Entities"
     };
 
-    private const int TOOL_PANEL_INTENTS = 0;
-    private const int TOOL_PANEL_ENTITIES = 1;
+    private readonly string[] toolPanelNamesWithoutAppInfo = new[]
```

Comment:
It's tempting to use a lambda to remove "Application" from toolPanelNames down below, instead of creating a duplicate list with one less entry. That creates two hardcoded occurrences of "Application", but this creates two hardcoded occurrences of the whole list. Or could turn these into consts; appInfoPanel etc